core/audits/bootup-time.js

```diff
@@ -109,6 +109,9 @@ class BootupTime extends Audit {
       settings.throttling.cpuSlowdownMultiplier : 1;
 
     const executionTimings = getExecutionTimingsByURL(tasks, networkRecords);
+    // Exclude our own tasks.
+    executionTimings.delete('_lighthouse-eval.js');
```

Comment:
Can we move this inside `getExecutionTImingsByURL`?

Reply:
> also, getExecutionTimingsByURL is in a `lib` folder, so we should try not to add overly lh-specific things there

Yes, this basically sums it up, but my opinion isn't strongly held. It's a balance between the expectation of `_lighthouse-eval.js` being filtered out and the expectation of all tasks in a period of a trace being accounted for. If there really was six seconds of a trace blocked by `evaluate` but it's filtered out, you might have trouble tracking down what's going on in the resulting audits, while if `_lighthouse-eval.js` isn't filtered out, it announces itself in the audit results.